.github/workflows/lint.yml

```diff
@@ -22,13 +22,11 @@ jobs:
         run: |
           # This is here to suppress a warning from GitHub
           git config --global --add safe.directory /__w/openscanhub/openscanhub
-          # TODO: This should be moved to `make lint` once we migrate to GitHub
-          REPO=origin
-          BRANCH=main
+
           # Check if we are running CI in a fork
           if ! git remote -v | grep -q 'origin.*openscanhub/openscanhub'; then
             REPO=openscanhub
             git remote add ${REPO} https://github.com/openscanhub/openscanhub.git
+            git fetch ${REPO}
           fi
-          git fetch ${REPO} ${BRANCH}
-          pre-commit run --show-diff-on-failure --color=always --from-ref ${REPO}/${BRANCH} --to-ref HEAD
+          make lint REPO=$REPO
```

Comment:
The CI job is now red.  Could it be related to this change?
```
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'diff', '--name-only', '--no-ext-diff', '-z', '/main..HEAD')
return code: 128
stdout: (none)
stderr:
    fatal: ambiguous argument '/main..HEAD': unknown revision or path not in the working tree.
    Use '--' to separate paths from revisions, like this:
    'git <command> [<revision>...] -- [<file>...]'
Check the log at /github/home/.cache/pre-commit/pre-commit.log
make: *** [Makefile:91: lint] Error 3
Error: Process completed with exit code 2.
```

Reply:
Addressed in 97b6caf.